src/shell/layout/tiling/mod.rs

```diff
@@ -4086,7 +4088,7 @@ where
                                     if render_active_child { 16 } else { 8 },
                                     alpha * if render_potential_group { 0.40 } else { 1.0 },
                                     output_scale,
-                                    GROUP_COLOR,
+                                    group_color(),
```

Comment:
So the `IcedElement` caches the theme, while the `render_output` function of the `TilingLayout` just locks the `THEME` many times through `*_color`-functions? (See overall review comment.)